src/pages/patientView/timeline2/timeline_helpers.tsx

```diff
@@ -47,8 +116,14 @@ export function configureTimelineToxicityColors(baseConfig: ITimelineConfig) {
                 '4': 'red',
             };
             return colorMap[grade.value] || POINT_COLOR;
+        } else {
```

Comment:
@inodb i think this is the key bit for coloring by track group.  so this can be merged.